docs/projects/blank/tutorial/1/what-it-does.md

```diff
@@ -0,0 +1,84 @@
+<%(TOC)%>
+
+# 1.3 - What the Blank Project does
+
+The Blank Project offers the basic ability to launch a deployment, connect a client-worker and server-worker to it, and let the Player Lifecycle module create a player entity to represent the client.
+
+On this page, we’ll walk you through this functionality, and introduce the tooling and workflows that you should become familiar with along the way.
+
+## Launch a local deployment
+
+First, hit `Ctrl+L` on Windows, or `Cmd+L` on Mac, to launch a local deployment. This opens a new terminal window and launches a local instance of SpatialOS using the `default_launch.json` launch configuration (located at the root of your SpatialOS project).
+
+> **Note: you can only have one instance of SpatialOS running on your local machine at a time.**
+
+<%(#Expandable title="What is inside default_launch.json?")%>
+
+This `default_launch.json` includes a load balancing configuration where SpatialOS expects at most one `UnityGameLogic` worker to be spawned in the world. The `manual_worker_connection_only` argument is specified to tell your local SpatialOS instance that you will manually connect a worker of that type to the deployment.
```

Comment:
```suggestion
This `default_launch.json` includes a load balancing configuration where SpatialOS expects at most one `UnityGameLogic` worker to be spawned in the world. 

The `manual_worker_connection_only` argument is specified to tell your local SpatialOS instance that you will manually connect a worker of that type to the deployment.
```

Think this will look a little cleaner